packages/vector_graphics/lib/vector_graphics.dart

```diff
@@ -110,7 +110,7 @@ class _VectorGraphicsWidgetState extends State<VectorGraphic> {
 
   @override
   void dispose() {
-    _pictureInfo?.picture.dispose();
+    _pictureInfo?.handle.layer = null;
```

Comment:
I _think_ this is the right way to use this....

Reply:
Yeah, but we probably just want a `dispose` method on `PictureInfo` to avoid leaking this implementation detail/letting some rando mess with it.

Reply:
fair

Reply:
done